README.md

```diff
@@ -4,6 +4,15 @@
 
 `scalastyle` is a configurable style linter for Scala code.
 
+### Building
+1. Install [sbt](http://www.scala-sbt.org/)
+2. Run `sbt docker`
+
+
+### Building release docker image
+1. Run `sbt assembly && cp target/scala-2.12/codeclimate-scalastyle-assembly-<version>.jar ./`.
+   This will create assembled jar with all dependencies.
+2. Run `docker build -t codeclimate/codeclimate-scalastyle .`    
```

Comment:
This is fine for now, but it would be great to have this step integrated within the docker image build process, maybe via a multi-stage Dockerfile: https://docs.docker.com/engine/userguide/eng-image/multistage-build

Reply:
As I understand SBT build tool is not supported thus this is the only workaround I could think of.

Reply:
There are [quite a few SBT images](https://hub.docker.com/search/?isAutomated=0&isOfficial=0&page=1&pullCount=0&q=sbt&starCount=0) out there. Docker definitely can run it. Or have I misunderstood your words?

Reply:
Oh, I get what you are saying. Basically use a docker image with sbt to build and assembly package and copy it into the final image. I'll do that.